### PR TITLE
Add new scripts to support automatic updating vesions of spring

### PR DESCRIPTION
**Pull request by Netyyyy**
As title

### CODE REVIEW

File: sdk/spring/scripts/README.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ Scripts in this folder is used to hold the scripts used in Spring Cloud Azure pr
...
-4. Update the comment at the beginning of `\eng\versioning\external_dependencies.txt`: Update the file name of `spring_boot_${SPRING_BOOT_VERSION}_managed_external_dependencies.txt`.
+4. Run command: `python .\sdk\spring\scripts\update_external_dependencies_comment.py -b ${SPRING_BOOT_VERSION}` to update the comment at the beginning of `\eng\versioning\external_dependencies.txt`.
 5. Run command `python .\eng\versioning\update_versions.py --ut external_dependency --sr`
 6. Run command `.\eng\versioning\pom_file_version_scanner.ps1 -Debug`. If there is error, fix it.
-7. Update changelog about compatible Spring Boot versions and Spring Cloud versions in `/sdk/spring/CHANGELOG.md`.
+7. Run command: `python .\sdk\spring\scripts\update_changelog.py -b ${SPRING_BOOT_VERSION} -c ${SPRING_CLOUD_VERSION}` to update changelog about Spring Boot and Spring Cloud dependencies versions in `/sdk/spring/CHANGELOG.md`.
 8. When generate `spring_boot_${SPRING_BOOT_VERSION}_managed_external_dependencies.txt` file, delete the old version file.
```
Comment (saragluna) [inline sdk/spring/scripts/README.md:16]: If we're going to keep this, why can't we use a script instead of the readme of each step?

**Review by saragluna — CHANGES_REQUESTED** [review]

File: sdk/spring/scripts/update_changelog.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    log.debug('Current working directory = {}.'.format(os.getcwd()))
+    args = get_args()
+    update_changelog(args.spring_boot_dependencies_version, args.spring_cloud_dependencies_version, CHANGE_LOG_FILE)
+    elapsed_time = time.time() - start_time
+    log.info('elapsed_time = {}'.format(elapsed_time))
+
+
+def change_to_root_dir():
+    os.chdir(os.path.dirname(os.path.realpath(__file__)))
+    os.chdir('../../..')
```
Comment (saragluna) [inline sdk/spring/scripts/update_changelog.py:55]: why do we need this

Comment (Netyyyy) [reply sdk/spring/scripts/update_changelog.py:55]: Change to `azure-sdk-for-java` directory

File: sdk/spring/scripts/update_changelog.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+
+import os
+import time
+import argparse
+
+from log import log
+
+
+CHANGE_LOG_FILE = 'sdk/spring/CHANGELOG.md'
```
Comment (saragluna) [inline sdk/spring/scripts/update_changelog.py:19]: CHNAGELOG_FILE

File: sdk/spring/scripts/update_changelog.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    start_time = time.time()
+    change_to_root_dir()
+    log.debug('Current working directory = {}.'.format(os.getcwd()))
+    args = get_args()
+    update_changelog(args.spring_boot_dependencies_version, args.spring_cloud_dependencies_version, CHANGE_LOG_FILE)
+    elapsed_time = time.time() - start_time
+    log.info('elapsed_time = {}'.format(elapsed_time))
+
+
+def change_to_root_dir():
```
Comment (saragluna) [inline sdk/spring/scripts/update_changelog.py:53]: change_to_repo_root_dir

**Review by saragluna — CHANGES_REQUESTED** [review]

**Review by saragluna — APPROVED** [review]

**Pull request merged**